docs/source/02_get_started/02_install.md

```diff
@@ -1,15 +1,15 @@
 # Install Kedro
 
-To install Kedro from the Python Package Index (PyPI) simply run:
+To install Kedro using `conda`:
 
 ```bash
-pip install kedro
+conda install -c conda-forge kedro
 ```
 
-It is also possible to install Kedro using `conda`:
+It is also possible to install Kedro without using `conda`, from the Python Package Index (PyPI):
 
 ```bash
-conda install -c conda-forge kedro
+pip install kedro
 ```
 
```

Comment:
I'm not sure of the intent here -- is there reason to swap the installation approaches around?

Reply:
From the Development Notes above: "Moved conda instructions above PyPI instructions, to be consistent with preference for conda in pre-requisites documentation."

Reply:
There's no problem doing `pip install xx` inside a conda environment, and pip packages are usually more widely available than that of conda forge, so I would keep the current order (pip -> conda) because this is a personal preference.

Reply:
OK, thanks for explaining. I have changed it back to pip then conda.